helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -78,6 +78,14 @@ spec:
             - /etc/chaos-daemon/cert/tls.crt
             - --key
             - /etc/chaos-daemon/cert/tls.key
+          {{- end }}
+            - --runtime-socket-path
+          {{- if eq .Values.chaosDaemon.runtime "docker" }}
+            - /host-run/docker.sock
```

Comment:
If the `socketPath` is set, we should also get the file name from it :thinking: .

Reply:
@YangKeao nice catch, updated